core/src/container.cpp

```diff
@@ -521,8 +529,9 @@ void SerialContainerPrivate::resolveInterface(InterfaceFlags expected) {
 		validateInterface<END_IF_MASK>(*last.pimpl(), expected);
 		// connect last child's (end) pull interface
 		if (const InterfacePtr& target = last.pimpl()->ends())
-			ends_.reset(new Interface(
-			    [this, target](Interface::iterator it, bool updated) { this->copyState(it, target, updated); }));
+			ends_.reset(new Interface([this, target](Interface::iterator it, bool updated) {
+				this->copyState<Interface::FORWARD>(it, target, updated);
```

Comment:
Copy-Paste-error? Shouldn't this read `Interface::BACKWARD`?

Reply:
Thanks, obviously a mistake.